pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -90,14 +136,15 @@ func (r *ReconcileClusterLogging) Reconcile(request reconcile.Request) (reconcil
 	if instance.Name != singletonName {
 		// TODO: update status
 
+		logrus.Infof("DBG_CL: Clusterlogging instance.Name != singletonName: '%s' != '%s'", instance.Name, singletonName)
 		return reconcile.Result{}, nil
 	}
 
 	if instance.Spec.ManagementState == loggingv1.ManagementStateUnmanaged {
 		return reconcile.Result{}, nil
 	}
 
-	if err = k8shandler.Reconcile(instance, r.client); err != nil {
+	if err = k8shandler.Reconcile(instance, r.client, false); err != nil {
```

Comment:
i think we should leave this function definition the same and instead have a separate call for the proxy config reconciler to just adjust the collector